server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -831,6 +832,43 @@ public void insert(InsertRowPlan insertRowPlan)
     }
   }
 
+  public void InsertSinglePoint(InsertSinglePointPlan insertSinglePointPlan)
```

Comment:
1. this function has too many duplicate codes with `insert()`;
2. Trigger related methods must also be maintained.